scripts/eslint-plugin-fig-linter/rules/no-missing-default-export.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  meta: {
+    type: "problem",
+  },
+  create(context) {
+    let hasExport = false;
+
+    return {
+      ExportDefaultDeclaration(node) {
+        hasExport = true;
+      },
+      'ExportSpecifier[exported.name="default"]'(node) {
+        hasExport = true;
+      },
+      "Program:exit"(node) {
+        if (!hasExport) {
+          context.report({
+            node,
+            message: "File must contain one default export.",
```

Comment:
The message can be a bit more expressive:
```suggestion
            message: "File must export default the completion spec object",
```